Base/home/anon/.config/LaunchServer.ini

```diff
@@ -24,6 +24,7 @@ pdf=/bin/PDFViewer
 profile=/bin/Profiler
 pp=/bin/PixelPaint
 directory=/bin/FileManager
+help=/bin/Help
```

Comment:
I think this is the wrong place, shouldn't this be under `[Protocol]`? This opens `.help` files.